src/pivot-table/components/containers/ScrollableContainer.tsx

```diff
@@ -1,26 +1,40 @@
 import React from "react";
-import type { Rect } from "../../../types/types";
 import { useBaseContext } from "../../contexts/BaseProvider";
 
+export enum ScrollableContainerOrigin {
+  LEFT_GRID = "leftGrid",
+  DATA_GRID = "dataGrid",
+  CONTAINER_GRID = "containerGrid",
+}
+
 interface ScrollableContainerProps {
-  rect: Rect;
+  width: number;
+  height: number;
   children: JSX.Element;
   onScroll: (e: React.SyntheticEvent) => void;
+  style?: React.CSSProperties;
+  showVerticalScrollbar?: boolean;
+  showHorizontalScrollbar?: boolean;
+  origin: ScrollableContainerOrigin;
 }
 
+const getTestId = (origin: ScrollableContainerOrigin) => `scrollable-container--${origin}`;
+
 const ScrollableContainer = (props: ScrollableContainerProps, ref: React.LegacyRef<HTMLDivElement>): JSX.Element => {
-  const { rect, children, onScroll } = props;
+  const { width, height, children, onScroll, style, showVerticalScrollbar, showHorizontalScrollbar, origin } = props;
   const { interactions } = useBaseContext();
 
   return (
     <div
       ref={ref}
-      data-testid="scrollable-container"
+      data-testid={getTestId(origin)}
```

Comment:
The data attribute should not be named "testid", if it's a value that we use for anything else then testing. Maybe just call it `data-key` instead?